core/main.go

```diff
@@ -90,13 +90,13 @@ func main() {
 	}
 
 	if *userTobeCreated != "" && *userofGroup != "" {
-		err := createConsumer(*userTobeCreated, *userofGroup, proxyBaseURL, EdgeXService, client)
+		err := createConsumer(*userTobeCreated, *userofGroup, proxyBaseURL, config.SecretService.SNIS, client)
```

Comment:
See comment above, this should be an independent property in the configuration.toml.